internal/util/connection.go

```diff
@@ -83,3 +84,11 @@ func (cc *ClusterConnection) GetIoctx(pool string) (*rados.IOContext, error) {
 
 	return ioctx, nil
 }
+
+func (cc *ClusterConnection) GetFSAdmin() (*ca.FSAdmin, error) {
```

Comment:
can we rename it to GetConnection makes more sense?

Reply:
connection would refer to `rados.Conn`, so I'd rather not do that